source/language/openpulse.rst

```diff
@@ -290,28 +298,262 @@ pushing the results into some buffer which is then accessed outside the program.
+    q3_π_half_sig = mix(π_half_sig, sine(q3_π_half_amp, q3_pos_freq-qubit_freq, 0.0, π_half_time));
+
+    for τ in [0: T]:
+        // Simultaneous π/2 pulses
+        play(eom_a_channel, constant(raman_a_amp, π_half_time) , raman_a_frame);
+        play(eom_b_channel, constant(raman_b_amp, π_half_time) , raman_b_frame);
+        play(aod_channel, q1_π_half_sig, q1_frame);
+        play(aod_channel, q1_π_half_sig, q2_frame);
+        play(aod_channel, q1_π_half_sig, q3_frame);
+
+        // Time delay all
+        delay(.*, τ/2)
+
+        // π pulse on qubit 1 only -- composed of two π/2 pulses
+        for _ in [0:1]:
+            play(eom_a_channel, constant(raman_a_amp, π_half_time) , raman_a_frame);
+            play(eom_b_channel, constant(raman_b_amp, π_half_time) , raman_b_frame);
+            play(aod_channel, q1_π_half_sig, q1_frame);
+
+        // Fence all then time delay
+        fence(.*)
+        delay(.*, τ/2)
+
+        // Phase shift the signals by a different amount -- or should I be shifting qubit_#_signal?
+        q1_frame.phase += tppi_1 * τ
+        q1_frame.phase += tppi_2 * τ
+        q1_frame.phase += tppi_3 * τ
+
+        // Simultaneous π/2 pulses
+        play(eom_a_channel, constant(raman_a_amp, π_half_time) , raman_a_frame);
+        play(eom_b_channel, constant(raman_b_amp, π_half_time) , raman_b_frame);
+        play(aod_channel, q1_π_half_sig, q1_frame);
+        play(aod_channel, q1_π_half_sig, q2_frame);
+        play(aod_channel, q1_π_half_sig, q3_frame);
+  }
+
+Multiplexed readout and capture
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example, we want to perform readout and capture of a pair of qubits, but mediated by a
+single physical tx and rx channel. The example is for just two qubits, but works the same for
+many (just adding more frames, waveforms, plays, and captures).
+
+.. code-block:: javascript
+
+  defcal multiplexed_readout_and_capture() $0 $1
+
+    // the tx/rx channel is the same for $0 and $1
+    tx_channel ro_tx = txch($0, "readout");
+    rx_channel ro_rx = rxch($0, "readout");
+
+    // readout frames of different frequencies
+    q0_frame = Frame(q0_ro_freq, 0); // time 0
+    q1_frame = Frame(q1_ro_freq, 0); // time 0
+
+    // flat-top readout waveforms
+    waveform q0_ro_wf = constant(amplitude=0.1, time=...);
+    waveform q1_ro_wf = constant(amplitude=0.2, time=...);
+
+    // multiplexed readout
+    play(ro_tx, q0_ro_wf, q0_frame);
+    play(ro_tx, q1_ro_wf, q1_frame);
+
+    // simple boxcar kernel
+    waveform ro_kernel = constant(amplitude=1, time=...);
+
+    // multiplexed capture
+    complex[32] q0_iqs = capture(ro_rx, q0_frame, ro_kernel);
+    complex[32] q1_iqs = capture(ro_rx, q1_frame, ro_kernel);
+
+
+Sample rate collisions
+-----------------------
+
+Incommensurate Rates
+~~~~~~~~~~~~~~~~~~~~
+
+Since the frame can be played on multiple channels, there may be an issue with sample rates.
```

Comment:
At the top, I believe you said `dt` is tied to the frame. But it seems here it's tied to the channel (which also certainly makes sense...)?